README.md

```
@@ -326,9 +447,26 @@ The Edge Function proxies GitHub API calls so your token never hits the client.
+### Nexus Spring of Code 2026 (NSoC'26)  
+### GirlScript Summer of Code 2026 (GSSoC'26)
+
+<img 
+  width="420"
+  alt="GSSoC & NSoC Banner"
+  src="https://github.com/user-attachments/assets/0cfe15fe-2d4a-4389-9a0b-8cf8d0cfd8cd"
+  style="border:4px solid #d4af37; border-radius:18px; padding:6px;"
+/>
+
+
+--- 
```

markdownlint-cli2 (0.22.1)

[warning] Horizontal rule style
Expected: ---; Actual: --- 

(MD035, hr-style)